### DISCUSSION TITLE
Syyskuun kuukausihuolto 11.9.

### DISCUSSION THREAD

**Discussion by CodoDerDritte** [KohaSuomi/Koha#1393] [Huolto- ja häiriötiedotteet]
Syyskuun huoltoikkunassa 11.9. klo 7-9 siirretään tuotantokohat uudelle, paremmin toimivalle tiedostojärjestelmälle, sekä muutetaan tiedostojärjestelmän konfiguraatiota. Muutoksista aiheutuu kaikissa kimpoissa katkoja huoltoikkunan aikana. Tunneloitujen automaattien yhteysosoitteet eivät huollossa muutu. Tiedotamme tässä ketjussa huollon etenemisestä.

**Comment by CodoDerDritte** [comment]
Huolto alkaa. Kohat menevät alas.

**Comment by CodoDerDritte** [comment]
Muut tuotannot Outia lukuun ottamatta ovat nyt käynnissä. Outi kiukuttelee. Selvitellään.

**Comment by CodoDerDritte** [comment]
Outikin nyt käynnissä. Käynnistellään nextit ja testit kunhan ehditään. Tuotantojen osalta katkot päättyvät.

**Comment by johannaraisa** [comment]
Nextit on nyt pystyssä.

**Comment by CodoDerDritte** [comment]
Testit nyt myös pystyssä.
